src/app/components/iot-gateway/iot-gateway.component.ts

```diff
@@ -339,10 +339,14 @@ export class IotGatewayComponent implements OnInit, AfterViewInit {
   }
 
   generateColor(gatewayName: string) {
-    let hash = 0;
-    for (var i = 0; i < gatewayName.length; i++) {
-      hash = gatewayName.charCodeAt(i) + ((hash << 5) - hash);
+    if (gatewayName == "HelloWorldGroup") {
```

Comment:
You might want to consider using `===` instead of `==`

https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons/359509#359509